src/org/usfirst/frc2175/commandmapper/JoystickEventMapper.java

```diff
@@ -27,7 +31,16 @@ public JoystickEventMapper(RobotConfig robotConfig,
         retractCatapult
                 .whenPressed(new RetractCatapultCommand(robotSubsystems));
 
+        JoystickButton runIntakeIn = gamepadConfig.getIntakeInButton();
+        runIntakeIn.whenPressed(new RunRollerbarIntakeInCommand(robotSubsystems,
+                intakeConfig.getRollerbarSpeedIn()));
```

Comment:
Use `RunIntakeInGroup` here instead of `RunRollerbarIntakeInCommand`. Same below for running it outward.